src/app/app.module.ts

```diff
@@ -77,6 +86,8 @@ import { StarComponent } from './shared/star/star.component';
     NoopAnimationsModule,
     MatSelectModule,
     RouterModule,
+    NgxStripeModule.forRoot('pk_test_51JUCrpATHQZZA29u2Z9T83a5jPQIPVHCa0wooyuyiaj561uWh4iNt9ZhocS0nSPWTft9rCbksrjq3Dk8pqKSLad100ZrRTjE9z'),
```

Comment:
Is this key safe to leave in git?

Reply:
Yes, this is the public key so it shouldn't be a problem, the private one is in secrets manager